src/js/hooks/useNavigation.jsx

```diff
@@ -0,0 +1,107 @@
+const useNavigation = ({
+  newRowFocusColumns,
+  existingRowFocusColumns,
+  tableData,
+  setFocusId,
+  setFocusIndex,
+  addEmptyRow,
+  productCode,
+  cycleCountId,
+}) => {
+  const getNextFocus = (columnId, rowIndex) => {
+    const currentIndex = newRowFocusColumns.indexOf(columnId);
+    const remainingColumns = newRowFocusColumns.slice(currentIndex + 1);
+    const newRowIndex = tableData[rowIndex + 1];
+
+    if (!remainingColumns.some(col => existingRowFocusColumns.includes(col))) {
```

Comment:
Did you run the linter here? I think it should, for example, change `col` to `(col)`